integration-tests/deployment/ccip/test_assertions.go

```diff
@@ -97,15 +108,28 @@ func ConfirmCommitWithExpectedSeqNumRange(
 			if len(report.Report.MerkleRoots) > 0 {
 				// Check the interval of sequence numbers and make sure it matches
 				// the expected range.
+				merkleFound := false
 				for _, mr := range report.Report.MerkleRoots {
 					if mr.SourceChainSelector == src.Selector &&
 						uint64(expectedSeqNumRange.Start()) == mr.MinSeqNr &&
 						uint64(expectedSeqNumRange.End()) == mr.MaxSeqNr {
-						t.Logf("Received commit report on selector %d from source selector %d expected seq nr range %s",
-							dest.Selector, src.Selector, expectedSeqNumRange.String())
-						return nil
+						t.Logf("Received commit report on selector %d from source selector %d expected seq nr range %s, token prices: %v",
+							dest.Selector, src.Selector, expectedSeqNumRange.String(), report.Report.PriceUpdates.TokenPriceUpdates)
+						merkleFound = true
+					}
+				}
+				tokenFound := false
+				// Check that LinkToken token price updates as expected
```

Comment:
I think this should be a separate assertion function - this one is specifically about ensuring a seq num was committed. Also It'd probably be simpler to just poll the PriceRegistry directly for a price update no?